src/TextInput2/TextInput2.tsx

```diff
@@ -141,13 +158,18 @@ const TextInput2: React.FC<Props> = ({
           />
         )}
       </div>
-      {!!helpText && <div className={cssClass.HELP_TEXT}>{helpText}</div>}
+      {(errorMessage || helpText) && (
+        <div className={classnames(cssClass.FOOTER_TEXT, errorMessage && cssClass.ERROR_MESSAGE)}>
```

Comment:
Minor nit: once again, not 100% sure of this but I think generally we'd like to add `aria-live` to elements that have an error message, especially since a screen reader would need to announce it even if the user's focus is no longer on the input. 

Basing this comment on [this W3 article](https://www.w3.org/WAI/tutorials/forms/notifications/#binary-messages), so feel free to ignore if it's not the case

Reply:
Thanks! I found `aria-errormessage`

Reply:
okay, I couldn't actually figure it out.  I will merge this for now then follow up with the #a11y channel